src/MICore/JsonLaunchOptions.cs

```diff
@@ -123,6 +124,33 @@ public abstract partial class BaseOptions
         public UnknownBreakpointHandling? UnknownBreakpointHandling { get; set; }
     }
 
+    internal class VisualizerFileConverter : JsonConverter
+    {
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            List<string> visualizerFile = new List<string>();
+            if (reader.TokenType == JsonToken.StartArray)
+            {
+                visualizerFile = serializer.Deserialize<List<string>>(reader);
+            }
+            else
```

Comment:
>else [](http://example.com/codeflow?start=12&length=4)

I wonder if we should check if the TokenType is a string so that if the user does something like `"visualizerFile": 12` they get a failure about bogus json.